draft-ietf-teep-protocol.md

```diff
@@ -709,6 +709,7 @@ Cons:
 
  - The Trusted Component Developer must host the Trusted Component Binary server
  - The device must fetch the Trusted Component Binary in another connection after receiving an Update message
+ - The device must reveal its IP address (and hence a clue as to its location), to the Trusted Component Binary server
```

Comment:
> The device must reveal eis location to the Trusted Binary server.

A server will be able to see a device's IP. But does a device "MUST" reveal? Should this be phrased as "A device has to reveal"?

Reply:
Updated